frontend/src/utils/constants.ts

```diff
@@ -3,6 +3,11 @@ export const KEYCLOAK = Object.freeze({
   REFRESH_TIME_MS: 10000,
 });
 
+export const IDENTITY_PROVIDERS = Object.freeze({
+  IDIR: 'idir',
+  BCEID_BASIC: 'bceid_basic',
```

Comment:
we'll need a third item:
`BCEID_BUSINESS: 'bceid_business'`
The field from the token we will use for identityId is also `bceid_user_guid`.

Reply:
actually.. i wonder if there needs to be this 3rd user type in the /app/config/idplist-local.json file.
and I believe we were going to name that file idplist.json
and line 34 and 35 of this file 
should be changed in the ap.ts to :
```js
async function readIdpList(): Promise<string> {
  const idplistDefaultPath = './config/idplist.json';
  const idplistOverridePath = './config/idplist-local.json';
```

Reply:
Omitting busines bceid for now until we get more clarity on upstream IDP behavior.